tests/php/Publisher/FilesystemPublisherTest.php

```diff
@@ -53,62 +53,62 @@ protected function setUp()
         $this->fsp = $mockFSP->setDestFolder('cache/testing/');
     }
 
-    protected function tearDown()
+    protected function tearDown(): void
     {
         if ($this->fsp !== null && file_exists($this->fsp->getDestPath())) {
             Filesystem::removeFolder($this->fsp->getDestPath());
         }
         parent::tearDown();
     }
 
-    public function testUrlToPathWithRelativeUrls()
+    public function testUrlToPathWithRelativeUrls(): void
     {
         $reflection = new \ReflectionClass(FilesystemPublisher::class);
         $urlToPath = $reflection->getMethod('URLtoPath');
         $urlToPath->setAccessible(true);
 
-        $this->assertSame(
+        $this->assertEquals(
```

Comment:
We should be using assertSame, not assertEquals

If there's broken tests, then update the $expected value rather than just using assertEquals